doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+
+!table id=lfrkeyparam caption=Key parameters of Lead-Cooled Fast Reactor (LFR) model
+| Parameter | Value  | Unit  |
+| :- | :- | :- |
+| Thermal power | $950$ | $MW$  |
+| Active core height | $105$  | $cm$ |
+| Number of fuel pins per assembly | $127$ | $-$ |
+| Gap thickness | $0.175$ | $mm$ |
+| Duct thickness | $3.5$ | $mm$ |
+| Fuel pin diameter | $10.7$ | $mm$ |
+| Fuel pin pitch | $13.3$ | $mm$ |
+| P/D | $1.24$ | $-$ |
+| Cladding thickness | $0.9$ | $mm$ |
+| Fuel pellet outer diameter | $8.55$ | $mm$ |
+| Fuel pellet inner diameter | $4.0$ | $mm$ |
+| Reynolds number | $51,000$ | $-$ |
+| Prandtl number | $0.0136$ | $-$ |
+| Peclet number | $705.21$ | $-$ |
+
+!media lfr/LFRgeometry.jpg
+       style=width:80%
+       id=lfrgeom
+       caption=Axial and radial geometry of the LFR single assembly model
+
+!media lfr/annularfuel.png
+       style=width:30%
+       id=annularfuel
+       caption=LFR annular fuel pin geometry
+
+## Cross Section Generation using MC2-3
+
+It is worth explaining the procedure to generate cross sections using MC2-3 for this model. MC2-3 is one of the cross section generation modules in Griffin. The axial leakage effect needs to be considered using the MC2-3/TWODANT two-step procedure for the generation of both homogeneous cross sections for non-fuel axial regions and heterogeneous region-wise cross sections for the fuel axial region. At this point of time, the MC2-3/TWODANT procedure is not fully available in Griffin due to a licensing issue with TWODANT. Once the feature is available in Griffin, the description below will be updated.
+
+### First step id=xs1st
+
+The first step is to generate a 1041-group `rzmflx`, which is the flux solution file of a TWODANT [!citep](TWODANT) SN calculation. It contains zone-wise Legendre moments of the angular flux solution up to a specified order. TWODANT is called inside MC2-3 when its executable path is specified by the `c_twodantexe` input parameter in the `library` block of an MC2-3 input with `l_twodant=T`. If TWODANT is not available, PARTISN [!citep](PARTISN) can be used alternatively since it creates a `rzmflx` file. `c_twodant_group=BG` means that the TWODANT transport calculation is performed using a group structure specified in `c_group_structure`. `c_geometry_type=mixture` means that homogeneous materials cooresponding to axial regions are used to generate macroscopic cross sections (in the 1041-group structure in this example) to be used in the TWODANT transport calculation. 
```

Comment:
What is the input data for this TWODANT calculation? The 1041-group library?

Reply:
The ISOTXS output file of the first MC2-3 calculation is the TWODANT input for cross sections. Let me add this description.

Reply:
I added more detailed description on the composition specification in the MC2-3 input. I also added the following paragraph.

```
The cross section set for the TWODANT calculation is the ISOTXS output file of the MC2-3 calculation in this first step. MC2-3 fills the matls option in the twodant block for the native TWODANT input with isotope names and atomic number densities taken from the material block. Thus, users do not need to take care of this part.
```